packages/vite/src/node/optimizer/scan.ts

```diff
@@ -134,7 +134,7 @@ function globEntries(pattern: string | string[], config: ResolvedConfig) {
 }
 
 const scriptModuleRE = /(<script\b[^>]*type\s*=\s*(?:"module"|'module')[^>]*>)(.*?)<\/script>/gims
-const scriptRE = /(<script\b[^>]*>)(.*?)<\/script>/gims
+const scriptRE = /(<script\b\s[^>]*>)(.*?)<\/script>/gims
```

Comment:
**issue:** I have tested it on https://regex101.com/ with 

```
<script-develop-pane/>
<script>var test = null</script>
<script/>
```

With `/(<script\b[^>]*>)(.*?)<\/script>/gims` the `script` with content `var test...` is detected
But with `/(<script\b\s[^>]*>)(.*?)<\/script>/gims` it's not
Is this wanted / did I misunderstood something? Or would that be a bug?

Reply:
done